test/test_cuda.py

```diff
@@ -1635,6 +1635,20 @@ def test_norm_type_conversion(self):
         a = torch.ones(65536).cuda().half()
         self.assertEqual(a.norm(p=0, dtype=torch.float32), 65536)
 
+    # Verifies that mem_get_info works, including when called for a different device
+    def test_mem_get_info(self):
+        def _test(idx):
+            before_free_bytes, before_available_bytes = torch.cuda.mem_get_info(idx)
+            t = torch.randn(1024 * 1024, device='cuda:' + str(idx))
```

Comment:
I vaguely remember one of our builds is not happy with `device='cuda:' + str(idx)`, but maybe I'm wrong